src/napari/view_layers.py

```diff
@@ -118,6 +122,29 @@ def _merge_layer_viewer_sigs_docs(func):
 _dims_params = Dims.__fields__
 
 
+def send_warning(
+    original_method: str, replaced_method: str, version: str = '0.7.0'
+):
+    """
+    Issue a deprecation warning for a method that will be removed in a future version.
+
+    Parameters
+    ----------
+    original_method : str
+        The name of the deprecated method (e.g., 'view_image').
+    replaced_method : str
+        The name of the recommended replacement method (e.g., 'add_image').
+    version : str, optional
+        The version in which the method will be removed. Default is '0.7.0'.
+    """
+    warnings.warn(
+        f"napari.{original_method} is deprecated and will be removed in 'napari' {version}.\n"
+        f'Use `viewer = napari.Viewer(); viewer.{replaced_method}(...)` instead.',
+        DeprecationWarning,
```

Comment:
```suggestion
        FututreWarning,
```

We use Future warning as deprecatin one is hidden from user by default python configurations